projects/showcase/src/app/demo/examples/geojson-line.component.ts

```diff
@@ -10,7 +10,7 @@ import { MapComponent, LayerComponent } from '@maplibre/ngx-maplibre-gl';
       "
       [zoom]="[15]"
       [center]="[-122.486052, 37.830348]"
-      [preserveDrawingBuffer]="true"
+      [canvasContextAttributes]="{preserveDrawingBuffer: true}"
```

Comment:
Perhaps we could add this as breaking change to the changelog?

Reply:
Sure, except we don't really follow semver since we mainly follow angular versioning...
I also changed the changelog editing to be more automated so that only when trying to release a new version the changelog will be updated as part of the "bump version PR" workflow.